LSTM.py

```diff
@@ -150,17 +152,21 @@ def train(self, vocab_idx, seq_len, batch_size, epochs, lr,
             stochastic.
             '''
             while len(possible_example_indices) > self.batch_size:
+                #Get #(batch_size) random training examples to take samples from
                 example_indices = random.sample(possible_example_indices, self.batch_size)
 
                 # Get processed data.
                 # print(len(possible_slice_starts[example_indices[0]]))
                 len_old = len(possible_example_indices)
+
+                #Are we removing stuff from (possible_slice_starts)? Not a global var
```

Comment:
It gets removed in the function as a pointer. it's not a global variable. it is passed to the function.